### PR TITLE
Move non-binary scripts into a scripts/ directory

### PR DESCRIPTION
**Pull request by chrisgilmerproj**
## Description

We have a lot of scripts and having a division between compiled binary files and bash scripts would be nice.  This PR moves non-binary scripts to `scripts/` and also puts `bin/` and `scripts/` on the `$PATH` to make these scripts easier to use.

## Reviewer Notes

Did I miss a reference?

## Setup

```sh
make clean
make build
```

## Code Review Verification Steps

* [x] Tested in the Experimental environment (for changes to containers, app startup, or connection to data stores)
* [x] Request review from a member of a different team.
* [x] Have the Pivotal acceptance criteria been met for this change?

## References

* [Pivotal story](https://www.pivotaltracker.com/story/show/164711660) for this change

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -21,36 +21,11 @@ __snapshots__
...
-/bin/save-fuel-price-data
-/bin/send-to-gex
-/bin/soda
-/bin/swagger
-/bin/tsp-award-queue
-/bin/webserver
-/bin/webserver_e2e
-
+# Downloaded certificates
 /bin/rds-combined-ca-bundle.pem
```
Comment (pjdufour-truss) [inline .gitignore]: This line could be removed too since it's under `/bin`

File: scripts/db-restore [hunk 2]
```diff
@@ -28,4 +28,4 @@ if [[ ! -f "$path" ]]; then
   exit 1
 fi
 
-psql -h localhost -U postgres -p "$port" -q -d "$db" < "$path" > /dev/null
+/usr/local/bin/psql -h localhost -U postgres -p "$port" -q -d "$db" < "$path" > /dev/null
```
Comment (pjdufour-truss) [inline scripts/db-restore]: It's not clear to me why we can't just use `psql`, since the wrapper was renamed.

Comment (chrisgilmerproj) [reply scripts/db-restore]: I'm considering reverting this.  But I usually prefer to be specific. Where it bites us is that the Docker file has this at `/usr/bin/psql` and brew installs on mac at `/usr/local/bin/psql`.

**Comment by chrisgilmerproj** [comment]
I would like to test this on experimental to ensure the deploy steps aren't messed up. And then I'll be ready to get this merged in.

**Comment by chrisgilmerproj** [comment]
I deployed this to Experimental and had zero problems. I think its ready to go!

**Pull request merged**